src/models/financialTaxes.js

```diff
@@ -0,0 +1,51 @@
+const taxFailureTypes = [
+  'File',
+  'Pay',
+  'Both',
+]
+
+const financialTaxes = {
+  Failure: {
+    presence: true,
+    hasValue: {
+      validator: {
+        inclusion: taxFailureTypes,
+      },
+    },
+  },
+  Year: {
+    presence: true,
+    date: {
+      requireMonth: false,
+      requireDay: false,
+    },
+  },
+  Reason: { presence: true, hasValue: true },
+  Agency: { presence: true, hasValue: true },
+  TaxType: { presence: true, hasValue: true },
+  Amount: {
+    presence: true,
+    hasValue: {
+      validator: {
+        numericality: {
+          greaterThan: 0,
+        },
+      },
+    },
+  },
+  Date: (value, attributes = {}) => {
+    const { DateNotApplicable } = attributes
+    if (DateNotApplicable && DateNotApplicable.applicable) {
+      return {
+        presence: true,
+        date: {
+          requireDay: false,
+        },
+      }
+    }
+    return {}
```

Comment:
Unable to get section to validate with date. It will only validate if I choose "Not applicable"